airbyte-config/init/src/main/java/io/airbyte/config/init/RemoteDefinitionsProvider.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.config.init;
+
+import static io.airbyte.config.init.PatchingHelpers.addMissingTombstoneField;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.util.MoreIterators;
+import io.airbyte.config.StandardDestinationDefinition;
+import io.airbyte.config.StandardSourceDefinition;
+import io.airbyte.config.persistence.ConfigNotFoundException;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpTimeoutException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This config persistence pulls the definition configurations from a remotely hosted catalog. It is
```

Comment:
This is no longer a `config persistence`

Reply:
Done in dff32bb35c8288f09660c5368b50d22e347f6d28